ui/package-lock.json

```diff
@@ -1,9609 +1,8 @@
 {
   "name": "fuse",
   "version": "0.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Just checking - did you add dependencies here with the version of npm in the `.nvmrc` file? I don't think it should should change the `lockfileVersion` unless an older version of npm was used